documentation/milestones/individual_milestones/static_site.md

```diff
@@ -0,0 +1,198 @@
+# Static Site for Grants Equity Project
+
+| Field           | Value                                                                |
+| --------------- | -------------------------------------------------------------------- |
+| Document Status | Accepted                                                             |
+| Epic Link       | [Issue 62](https://github.com/HHS/grants-equity/issues/62)           |
+| Epic Dashboard  | [Milestone Roadmap](https://github.com/orgs/HHS/projects/12/views/4) |
+| Target Release  | 2023-09-20                                                           |
+| Product Owner   | Lucas Brown                                                          |
+| Document Owner  | Billy Daly                                                           |
+| Lead Developer  | Aaron Couch                                                          |
+| Lead Designer   | Andy Cochran                                                         |
+
+## Short description
+
+Launch a simple site at beta.grants.gov that provides static, informational content about the Grants Equity initiative, which includes both the Grants.gov Modernization and NOFO Transformation workstreams.
+
+## Goals
+
+### Business description & value
+
+The launch of a static site for the Grants Equity project represents the culmination of multiple internally focused deliverables and serves as a landing page where key stakeholders can access information about the project. 
+
+By sharing this information in a publicly accessible format and investing early in the infrastructure used to host it, this milestone aims to demonstrate the following value propositions:
+
+- Raises awareness about the Grants Equity project, its priorities, and ongoing workstreams
+- Establishes beta.grants.gov as the primary location that stakeholders can visit for project updates and previews of deliverables from each of the workstreams
+- Proves the succesful completion of technical milestones that enable faster development without sacrificing code quality or security
+- Delivers an early win that both internal and external stakeholders can rally around, which helps build momentum and enthusiam for the project
+
```

Comment:
Suggestion: Do we want to add a bullet around the parallel approach allows us quickly deliver to users and migrate users and functionality in phases and iteratively?

Reply:
Great point and a valuable addition! I added it in [this commit](https://github.com/HHS/grants-equity/pull/243/commits/8f8cd6ac6e0d30811ccbd122b1734aac043e5133)